docs/src/resources/two-side-column.md

```diff
@@ -0,0 +1,923 @@
+<!Doctype html>
+<html>
+<title> Two side column </title>
+
+```scala mdoc:invisible
+import Chisel.Queue
+import chisel3._
+import chisel3.util.{DecoupledIO, switch, is}
+import chisel3.stage.ChiselStage
+import chisel3.experimental.ChiselEnum
+import chisel3.util.{Cat, Fill, DecoupledIO}
+```
+
+<body>
+    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
+    <table border ="0">
+        <h1>Creating a Module</h1>
+        <tr>
+            <td><b style="font-size:30px">Verilog</b></td>
+            <td><b style="font-size:30px">Chisel</b></td>
+         </tr>
+         <tr>
+<td>
+
+```
+module Foo (
+  input  a,
+  output b
+);
+  assign b = a;
+endmodule
+```
+
+</td>
+    <td>
+
+```scala mdoc
+class Foo extends Module {
+  val a = Input(Bool())
+  val b = Output(Bool())
+  b := a
+}
+```
+</td>
+         </tr>
+    <tr>
+<td>
+
+```
+module PassthroughGenerator(
+  input        clock,
+  input        reset,
+  input  [9:0] io_in,
+  output [9:0] io_out
+);
+  assign io_out = io_in;
+endmodule
+```
+</td>
+<td>
+
+```scala mdoc:silent
+class PassthroughGenerator(width: Int) extends Module {
```

Comment:
This example shows parameterization, but we don't show it in the verilog. What do we want to be showing with this example?

Reply:
My suggestion would be to move this example down, before the parameterized width adder example which is more complicated, then parameterize the verilog accordingly